compiler/resolution/functionResolution.cpp

```diff
@@ -11746,14 +11747,54 @@ static void postResolveLiftStaticVars() {
       // Move the definition point of the wrapper into the module scope.
       // But first, keep a handle on the block to be able to move it later.
       auto wrapperBlock = toBlockStmt(wrapperSym->defPoint->parentExpr);
-      INT_ASSERT(wrapperBlock);
-      call->getModule()->block->insertAtHead(wrapperSym->defPoint->remove());
+      targetMod->block->insertAtHead(wrapperSym->defPoint->remove());
+
+      // If we created some cleanup code to run in the per-locale case,
```

Comment:
Can we point devs towards where it's created so that people can reference that mega-comment?

Reply:
Yes, I've moved the comment about normalize.cpp up to this point.